.github/workflows/continuous-integration.yaml

```diff
@@ -87,11 +87,8 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
-          - 10
-          - 12
-          - 14
-          - 16
           - 18
+          - 20
```

Comment:
@gr2m
maybe also adding node 21 to detect potential issues before node 22 gets released.